app/api/admin.py

```
@@ -0,0 +1,856 @@
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND, detail="User not found"
+        )
+
+    user["id"] = str(user["_id"])
+    return UserResponse(**user)
+
+
+@router.put("/users/{user_id}/deactivate")
+async def deactivate_user(
+    user_id: str,
+    current_admin: dict = Depends(get_current_admin_user),
```

Ruff (0.13.2)

Unused function argument: `current_admin`

(ARG001)